subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -16,6 +16,14 @@
                 "Abstract method has been added in implemented interface"
```

Comment:
Does this violation still belong? Or shall we fix it in a separate PR?

Reply:
Yes, both of these will show up. I didn't want to add extra logic to make these mutually exclusive, and I don't think it's worth it right now.